internal/provider/resource/project_role_resource.go

```diff
@@ -92,9 +116,41 @@ func (r *projectRoleResource) Schema(_ context.Context, _ resource.SchemaRequest
 				Computed:      true,
 				PlanModifiers: []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
 			},
+			"permissions_v2": schema.SetNestedAttribute{
+				Optional:    true,
+				Description: "The permissions assigned to the project role. Refer to the documentation here https://infisical.com/docs/internals/permissions for its usage.",
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"action": schema.SetAttribute{
+							ElementType: types.StringType,
+							Description: "Describe what actions an entity can take.",
+							Required:    true,
+						},
+						"subject": schema.StringAttribute{
+							Description: "Describe the entity the permission pertains to.",
+							Required:    true,
+						},
+						"inverted": schema.BoolAttribute{
+							Description: "Whether rule forbids. Set this to true if permission forbids.",
+							Required:    true,
```

Comment:
i'm thinking we don't have to require this and can default it to false. what do you think?

Reply:
the problem with this one is we can't set a default value for it without making it into a`computed` field... and we can't make it into a computed field because it's inside a `Set`